pubmed_dl/ncbi.py

```diff
@@ -112,28 +113,31 @@ def uids_to_docs(uids: List[str]) -> List[Dict[str, Collection[Any]]]:
         upper = min([lower + MAX_EFETCH_RETMAX, num_uids])
         id = uids[lower:upper]
         try:
+            if "bruh" in id:
+                raise e
             eutil_response = _get_eutil_records("efetch", id, rettype="medline", retmode="text")

```

Comment:
`_get_eutil_records` is the function that will tend to fail, if anything. Right now it's passing along the error but we could change it to continue on e.g. literally log error and `continue`